ops/fhir_validations/filter_json_response.py

```diff
@@ -0,0 +1,240 @@
+import json
+import sys
+
```

Comment:
Should the script name be less general since this is specific to the spike and hard codes the filters to be applied?

Reply:
sure, I can rename it to make it clear.